src/GCodes/GCodes3.cpp

```diff
@@ -184,6 +184,9 @@ GCodeResult GCodes::GetSetWorkplaceCoordinates(GCodeBuffer& gb, const StringRef&
 		if (seen)
 		{
 			reprap.MoveUpdated();
+			String<GCODE_LENGTH> scratch;
```

Comment:
Use a shorter string e.g. StringLength100. Suggest you define a constant StringLengthLoggedCommand for this type of situation (and set it equal to StringLength100).

Reply:
Done.